server/api/offline_batch_results.py

```diff
@@ -310,6 +310,33 @@ def finalize_offline_batch_results(
     return jsonify(status="ok")
 
 
+@api.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/round/<round_id>/results/batch/finalize",
+    methods=["DELETE"],
```

Comment:
bold move, HTTP friend.

Reply:
really these endpoints should have ended in `/finalizedAt`